docs/src/modules/components/PersistScroll.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+let savedScrollTop = null;
+
+export default function PersistScroll(props) {
```

Comment:
This should live unexported inside Appdrawer.js because it uses a singleton. Otherwise it implies I can use this for other scrollbars as well.

Reply:
This should live unexported inside Appdrawer.js because it uses a singleton. Otherwise it implies I can use this for other scrollbars as well.

Reply:
Ok